ui/desktop/electron-app/config/forge.config.js

```diff
@@ -39,7 +42,7 @@ module.exports = {
         done();
       },
     ],
-    extraResource: ['./cli'],
+    extraResource,
```

Comment:
I though on leaving the `beforeAsar` regardless, so we make sure we always clean the CLI directory and we do not pack it by error. Let me know if you will like to discuss this.

Reply:
Yeah it shouldn't matter much, we prevent errors if the path doesn't exist with `force: true`